README.md

```diff
@@ -73,14 +73,16 @@ and processor architecture listed below.
 
 ### Supported operating system versions
 
+* [CentOS Linux] 7
 * [Debian] 11
 * [Docker]
 * [FreeBSD] 13.0
-* [Ubuntu] 20.04
-* [Rocky Linux] 8.4
+* [Ubuntu] 22.04
+* [Rocky Linux] 8.5
```

Comment:
I'm using Rocky Linux 8.6.

This needs to be changed in multiple places.

Reply:
Fixed.